spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/config/StandardizationConfig.scala

```diff
@@ -31,9 +31,10 @@ import scala.util.Try
  */
 case class StandardizationConfig(rawFormat: String = "xml",
                                  charset: Option[String] = None,
+                                 nullValue: Option[String] = None,
                                  rowTag: Option[String] = None,
                                  csvDelimiter: Option[String] = None,
-                                 csvHeader: Option[Boolean] = Some(false),
+                                 csvHeader: Option[Boolean] = None,
```

Comment:
This, together the check further down, allowed `--header false` to be specified for other then `csv` formats too.